ttnn/cpp/ttnn/operations/pool/generic/device/kernels/dataflow/reader_pool_2d_multi_core_sharded_with_halo_wide.cpp

```diff
@@ -65,44 +64,40 @@ void kernel_main() {
     constexpr uint32_t in_reader_indices_cb_id = get_compile_time_arg_val(19);
     constexpr uint32_t in_scalar_cb_id = get_compile_time_arg_val(20);
 
-    constexpr uint32_t ROW_HW = 64;
-
     // Reduce scalar = 1
     if (reader_id == 0) {
         cb_reserve_back(in_scalar_cb_id, 1);
-        uint32_t bf16_one_u16 = bf16_one_u32 >> 16;
-        // fill 1 row w/ scalar
-        fill_with_val(get_write_ptr(in_scalar_cb_id), ROW_HW, bf16_one_u16);
+        fill_with_val(get_write_ptr(in_scalar_cb_id), TILE_WIDTH, bf16_scalar >> 16);
```

Comment:
I'm not getting why are we now writing less data here, 
we used to use ROW_HW = 64 and now it's TILE_WIDTH = 32

Reply:
Ok if everything works with smaller value I guess that is good :)